source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb

```diff
@@ -49,11 +49,15 @@ If __Test Mode__ is at the bottom of the left-hand menu, switch to __Production
 
 10. Enter your XML password into the __Password__ field in your GOV.UK Pay account.
 
-11. In the __Payment service__ section of your Worldpay account, set __Capture delay (days)__ to __Off__ (not __0__).
+11. Go to the __Payment service__ section of your Worldpay account.
+
+12. Set __Capture delay (days)__ to __Off__ (not __0__).
 
     If you do not do this, you may take money from your users before they've confirmed their payment.
 
-12. In your GOV.UK Pay account, select __Save credentials__.
+13. Set __Maximum Transaction Amount__ to the highest payment you will take. The maximum amount is 10,000,000 pence (£100,000).
```

Comment:
I think that they need to request that Worldpay do this for them, I don't think it's editable. Do we need to specify that? Or can they usually edit it themselves?

Reply:
Sorry, you're right! I misread the thread. I've fixed in https://github.com/alphagov/pay-tech-docs/pull/393/commits/8cc2ec95c56255e3948d6e6d0235549b49b2babd.